src/geneweaver/client/gedb.py

```diff
@@ -61,7 +67,9 @@ class DataResult:
     weights: List[float] = None  # noqa: N815
     geneIds: List[str] = None  # noqa: N815
     strains: List[str] = None  # noqa: N815
+    strain: str = None  # noqa: N815
```

Comment:
What's the differtence between the singular and plural `strain`/`strains`?

Reply:
The type of search.